src/components/tree-item/tree-item.component.ts

```diff
@@ -257,7 +258,7 @@ export default class SlTreeItem extends ShoelaceElement {
             })}
             aria-hidden="true"
           >
-            ${when(this.loading, () => html` <sl-spinner></sl-spinner> `)}
+            ${when(this.loading, () => html` <sl-spinner part="spinner"></sl-spinner> `)}
```

Comment:
Thanks for the PR!

When exposing a part, we want to make sure we expose "sub-parts" too, so this needs something like `exportsparts="base:spinner__base"` [per these guidelines](https://shoelace.style/resources/contributing#naming-css-parts) and another `@csspart` entry.

For reference, [spinner parts can be found here](https://shoelace.style/components/spinner/#parts).

Reply:
Added!